README.md

```diff
@@ -13,8 +13,9 @@ The Docker Image can be installed in either of the two ways:
 
 - Pull the image (Beginners)
     ```bash
-    docker pull sakshaymahna/ros-turtlebot-tutorial:noetic
+    docker pull {GITHUB USERNAME}/ros-turtlebot-tutorial:noetic
```

Comment:
Actually, the `sakshaymahna` over here, refers to the Docker Hub repository. The Docker Image we are using is uploaded on Docker Hub. Since, it's present on my account, we need to use `sakshaymahna/ros-turtlebot-tutorial`. 

For instance, this (https://hub.docker.com/r/google/mysql) is the DockerHub repo for MySQL by Google. In order to pull it, we need to use the `google/mysql`, since it's on Google's Docker Hub.

Reply:
Got it. Let's specify that in the README then so that people don't misunderstand it as a variable that needs to be modified. Was a bit misleading without the explanation.